src/move/econia/sources/user.move

```diff
@@ -409,6 +409,42 @@ module econia::user {
         size: u64
     }
 
+    /// Table keys are market account IDs.
+    struct MarketEventHandles has key {
+        fill_events: Table<u128, EventHandle<FillEvent>>,
+        limit_order_events: Table<u128, EventHandle<LimitOrderEvent>>
+    }
+
+    struct FillEvent has copy, drop, store {
+        market_id: u64,
+        size: u64,
+        price: u64,
+        maker_side: bool,
+        maker: address,
+        maker_custodian_id: u64,
+        maker_order_id: u128,
+        taker: address,
+        taker_custodian_id: u64,
+        taker_order_id: u128
+    }
+
+    struct LimitOrderEvent has copy, drop, store {
+        market_id: u64,
+        user: address,
+        custodian_id: u64,
+        integrator: address,
+        side: bool,
+        size: u64,
+        price: u64,
+        restriction: u8,
+        self_match_behavior: u8,
+        base_traded_as_taker: u64,
+        quote_traded_as_taker: u64,
+        fees_as_taker: u64,
+        size_posted_as_maker: u64,
+        order_id: u128
+    }
```

Comment:
Is the idea that this event will be emitted only when a limit order is placed? Or will cancellations and modifications be variations of this? Either way, the name should be changed to make its role more clear.

Reply:
58a7aa9 renames to `PlaceLimitOrderEvent`